db/migrate/20180913111742_add_muted_at_to_notifications.rb

```diff
@@ -0,0 +1,6 @@
+class AddMutedAtToNotifications < ActiveRecord::Migration[5.2]
+  def change
+    add_column :notifications, :muted_at, :datetime, null: true
+    add_index :notifications, :muted_at
+  end
+end
```

Comment:
Can you commit the change to `db/schema.rb` that this migration generates as well

Reply:
sure. will do that.